Makefile

```diff
@@ -2,6 +2,7 @@ PREFIX       ?= /usr/local
 LOCALEPREFIX ?= $(PREFIX)/share/locale
 MANPREFIX    ?= $(PREFIX)/share/man
 PANDOC       ?= $(shell which pandoc)
+DOWNGRADE_CONF ?= /etc/xdg/downgrade/downgrade.conf
```

Comment:
I think this needs to respect `DESTDIR`. Otherwise, when used in a `PKGBUILD`, it would try to install onto the system _building_ the package, not the system _installing_ the package.